Finish book challenge but it's weird

sasianan19 opened the pull request.

waldenwang9966 commented on Sierra/challenge_14/book_challenge.py:
```diff
@@ -0,0 +1,120 @@
+Then, print out the books attribute of nyt_bestsellers
+'''
+# This doesn't work here but it works in the book_list_class.py script!
+#print(nyt_bestsellers.books)
```
For this part, you are supposed to instantiate the booklist objects first, and then add books to it, and then print them out. So basically, there are three things you have to write in this section.

sasianan19 replied:
It says to do those steps in the book_list_class.py script so I thought I was supposed to do it in there /: that's where I did those steps

waldenwang9966 commented on Sierra/challenge_14/book_list_class.py:
```diff
@@ -0,0 +1,37 @@
+	def remove_title(self, title):
+		"""Remove a book from the book list"""
+		if title in self.books:
+			self.books.remove(title)
```
For the original design, you can loop through all the books, and then call  if book.title == title, then remove the book object. Something like this:
for book in self.books:
    if book.title == title: 
         self.book.remove(book)

waldenwang9966 commented on Sierra/challenge_14/book_list_class.py:
```diff
@@ -0,0 +1,37 @@
+
+	def display_titles(self):
+		"""Print out all titles currently in the book list"""
+		pass
```
Here, what you can do is to initialize an empty titles list, and then append book one by one, and then sort the titles using sort function.

sasianan19 commented:
For the most recent commit, I fixed what I could but because of the way I added the title and author, my other methods in the book_list_class.py don't work the way they should. I understand how the logic is supposed to work, but when I tried the ways it's supposed to be written, they didn't work for mine. Idk if this makes sense, but I hope so lol.
